client/package.json

```diff
@@ -824,7 +824,7 @@
     "test:grammar": "vscode-tmgrammar-test ./test/grammars/test-cases/*.bb",
     "snap-grammar": "vscode-tmgrammar-snap ./test/grammars/snaps/*.bb -u",
     "vscode:prepublish": "npm install --omit=dev && cd ../server && npm install --omit=dev && npm run installServer",
-    "package": "npm run vscode:prepublish && vsce package --target linux-x64 && vsce package --target linux-arm64 && vsce package --target linux-armhf && vsce package --target win32-x64 && vsce package --target win32-arm64 && vsce package --target web"
+    "package": "npm run vscode:prepublish && vsce package --target linux-x64 && vsce package --target linux-arm64 && vsce package --target linux-armhf && vsce package --target win32-x64 && vsce package --target win32-arm64 && vsce package --target web && vsce package --target darwin-x64"
```

Comment:
Also add darwin-arm64. It should work out of the box since nothing is compiled.

Reply:
We could also add alpine-x64, alpine-arm64. There's no reason they wouldn't work.